app/controllers/draft_vaccination_records_controller.rb

```diff
@@ -266,4 +270,22 @@ def vaccine_criteria
       without_gelatine:
     )
   end
+
+  def create_triage_record_for_next_mmr_vaccination
+    return if @patient.triages.delay_vaccination.exists?
+
+    delay_date = MINIMUM_MMR_DOSE_INTERVAL_DAYS.days.from_now.to_date
+    TriageForm.new(
```

Comment:
Do we need to go through the form, could we call `Triage.create!` directly?

Reply:
I initially thought about using `Triage.create!` but looking at the `TriageForm`, I got the impression there's more logic in there and could be leveraged implicitly, but happy to use `Triage.create!`.